translations/round_structure.tex/pl.po

```diff
@@ -80,6 +80,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Rozgrywka podzielona jest na Rundy, podczas których każdy gracz wykonuje swoją Turę w kolejności zgodnej z ruchem wskazówek zegara, zaczynając od gracza rozpoczynającego grę.\n"
+"W swoich Turach gracze przemieszczają swoich \\hyperlink{Heroes}{Bohaterów} na Mapie Gry, budują nowe Budynki w swoich \\hyperlink{Town}{Miastach} i rekrutują \\hyperlink{Units}{Jednostki}, próbując spełnić warunek zwycięstwa Scenariusza.\\par\n"
+"\\textbf{Na początku każdej Rundy}, z wyjątkiem pierwszej, wykonaj następujące kroki :\n"
+"\\begin{itemize}\n"
+" \\item Obróć na aktywną stronę wszystkie użyte Żetony Budowy, Populacji i/lub Księgi Zaklęć.\n"
+" \\item Obróć na aktywną, zieloną stronę użyte Żetony Ruchu (PR).\n"
```

Comment:
```suggestion
" \\item Obróć na aktywną, zieloną stronę użyte Żetony Punktów Ruchu (PR).\n"
```
Może lepiej wyraźnie to zaznaczyć?

Reply:
Tu akurat miałem dylemat, ale polska instrukcja, strona 5 opisuje je jako Żetony Ruchu

Reply:
Z tym że później w instrukcji używany jest zwrot PR. Strona 11, sekcja "Bohater Pomocniczy"

Reply:
Ech te problemy pierwszego świata :|

Reply:
A może tak: "Żetony Ruchu (Punkty Ruchu - PR)"

Reply:
Prawdę mówiąc nazywanie ich żetonami ruchu imo jest błędne, po prostu zignorowałbym ten wpis z instrukcji i trzymał się skrótu, który sugeruje nazywać je Punktami Ruchu - a ich znaczniki Żetonami Punktów Ruchu

Reply:
Dobra, niech będzie.